recipes/argutils/LICENSE.txt

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Is there an upstream issue about the missing license file?

Reply:
The license file unfortunately missing in the upstream repo. The license is only stated in the setup.py file.

Reply:
Filed an upstream issue about the missing license file: eclarke/argutils/issues/2